user_test.go

```diff
@@ -30,31 +30,41 @@ func TestClient_MuteUser(t *testing.T) {
 
 	user := randomUser()
 
-	err := c.MuteUser(user.ID, serverUser.ID)
+	err := c.MuteUser(user.ID, serverUser.ID, map[string]interface{}{"timeout": 60})
 	require.NoError(t, err, "mute user")
 
 	users, err := c.QueryUsers(&QueryOption{
 		Filter: map[string]interface{}{
 			"id": map[string]string{"$eq": serverUser.ID},
 		}})
-
 	require.NoError(t, err, "query users")
 
 	assert.Lenf(t, users[0].Mutes, 1, "user mutes exists: %+v", users[0])
 
 	mute := users[0].Mutes[0]
 	assert.NotEmpty(t, mute.User, "mute has user")
 	assert.NotEmpty(t, mute.Target, "mute has target")
+	assert.NotEmpty(t, mute.Expires, "mute has expires")
 }
 
 func TestClient_MuteUsers(t *testing.T) {
 	c := initClient(t)
 	initChannel(t, c)
 
-	users := []string{randomUser().ID, randomUser().ID}
+	usernames := []string{randomUser().ID, randomUser().ID}
```

Comment:
this randomness can cause issues in tests once in a while, plug in some specific users